### ISSUE TITLE
Azure Communication Service - unable to run the quickstart app in Azure App Service

### ISSUE THREAD

**Issue by prawin2k** [Azure-Samples/communication-services-web-calling-tutorial#37]
i have deployed the app into Azure App Service. 

It's working fine locally. However, when I deploy it into Azure App Service it fails with a 404 error. Please see below the error.

![YIqPe 1](https://user-images.githubusercontent.com/10738750/127740335-b9bd90d6-1edf-4a50-81fe-574053d0fb14.png)

In the section, **Code Structure** it says

> ./webpack.config.js: Project bundler. Has a simple local server for user token provisioning.

Does it mean it doesn't work after deployment to Production? If not, then how do I get it work?

**Comment by t-sanderv** [comment]
Yes it does, because it relies on a local development server that only works when running `npm run start`. I have a fix in a fork [here](https://github.com/t-sanderv/-communication-services-web-calling-tutorial-fork), I will work on merging this fork into this repo.
